server/webapp/WEB-INF/rails.new/spec/presenters/api_v2/config/property_config_representer_spec.rb

```diff
@@ -0,0 +1,71 @@
+##########################################################################
+# Copyright 2016 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+
+require 'spec_helper'
+
+describe ApiV2::Config::PropertyConfigRepresenter do
+  it 'should serialize property' do
+    property = com.thoughtworks.go.config.ArtifactPropertiesGenerator.new('foo', 'target/emma/coverage.xml', 'substring-before(//report/data/all/coverage[starts-with(@type,class)]/@value, %)')
+    presenter = ApiV2::Config::PropertyConfigRepresenter.new(property)
+    actual_json = presenter.to_hash(url_builder: UrlBuilder.new)
+
+    expect(actual_json).to eq(property_hash)
+  end
+
+  it 'should deserialize test artifact' do
+    expected = com.thoughtworks.go.config.ArtifactPropertiesGenerator.new
+    expected.setName('foo')
+    expected.setSrc('target/emma/coverage.xml')
+    expected.setXpath('substring-before(//report/data/all/coverage[starts-with(@type,class)]/@value, %)')
+
+    actual    = com.thoughtworks.go.config.ArtifactPropertiesGenerator.new
+    presenter = ApiV2::Config::PropertyConfigRepresenter.new(actual)
+    presenter.from_hash(property_hash)
+    expect(actual.getSrc).to eq(expected.getSrc)
+    expect(actual.getName).to eq(expected.getName)
+    expect(actual.getXpath).to eq(expected.getXpath)
+    expect(actual).to eq(expected)
+  end
+
```

Comment:
Can you refactor the above two tests a bit, seems like you can have the ```property``` object in before each, remove the expected object in the second test and do,

```
expect(actual).to eq(@property)
```